app/controllers/volunteer_ops_controller.rb

```diff
@@ -57,15 +69,15 @@ def volunteer_op_params
 
   private
 
-  def build_map_markers(organisations)
-    ::MapMarkerJson.build(organisations) do |org, marker|
+  def build_map_markers(organisations, type = :harrow, include_extra_organisation_data = true)
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Metrics/MethodLength: Method has too many lines. [15/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)